src/platform_impl/linux/x11/window.rs

```diff
@@ -1306,6 +1306,40 @@ impl UnownedWindow {
         self.set_ime_position_physical(x, y);
     }
 
+    fn set_focus_inner(&self) -> util::Flusher<'_> {
+        unsafe {
+            let atom = self.xconn.get_atom_unchecked(b"_NET_ACTIVE_WINDOW\0");
+            self.xconn.send_client_msg(
+                self.xwindow,
+                self.root,
+                atom,
+                Some(ffi::SubstructureRedirectMask | ffi::SubstructureNotifyMask),
+                [1, ffi::CurrentTime as c_long, 0, 0, 0],
+            )
+        }
+    }
```

Comment:
This code is dead? If the goal was to call this with the `focus_window_inner` call, I don't see why it wouldn't just be inlined?